### ISSUE TITLE
"del" is not highlighted as a keyword

### ISSUE THREAD

**Issue by dactylo** [MattDMo/PythonImproved#42]
It's still a keyword in Python 3, right?

**Comment by MattDMo** [comment]
```python
# Python 3.4.3
In [10]: import keyword

In [11]: from pprint import pprint as pp

In [12]: pp(keyword.kwlist)
['False',
 'None',
 'True',
 'and',
 'as',
 'assert',
 'break',
 'class',
 'continue',
 'def',
 'del',
 'elif',
 'else',
 'except',
 'finally',
 'for',
 'from',
 'global',
 'if',
 'import',
 'in',
 'is',
 'lambda',
 'nonlocal',
 'not',
 'or',
 'pass',
 'raise',
 'return',
 'try',
 'while',
 'with',
 'yield']
```
You are correct. Currently, `del` is scoped as `support.function.builtin.python`, but it is not a [builtin](https://docs.python.org/3.4/library/functions.html#built-in-funcs). I'm preparing some other changes for the next release, so I'll update this as well. Thanks for the heads up!

**Comment by cknv** [comment]
I think it was fixed already in https://github.com/MattDMo/PythonImproved/commit/8d23d3cbf5db5c111617d01da89a26b90b2682a4.

**Comment by MattDMo** [comment]
Yes, I found that after I wrote the last message. It's in the `unicode` branch, which I'll be merging with `master` when I'm done testing it. Thanks!

**Comment by cknv** [comment]
I think it's already in the `master` branch. I found the commit in the history on `master` :)

It [seems to have hit](https://github.com/MattDMo/PythonImproved/network) `master` just before the `unicode` branch was made.

**Comment by MattDMo** [comment]
@cknv apparently you know my codebase better than I do :)

**Comment by MattDMo** [comment]
This has been fixed in the `unicode` branch, and will be released in the upcoming version 2.0.
